modules/account/src/main/kotlin/edu/stanford/spezi/module/account/register/RegisterFormValidator.kt

```diff
@@ -1,44 +1,14 @@
 package edu.stanford.spezi.module.account.register
 
-import java.time.LocalDate
 import javax.inject.Inject
 
 internal class RegisterFormValidator @Inject constructor() : FormValidator() {
```

Comment:
this and `LoginFormValidator` aren't needed anymore, we can make `FormValidator` injectable and use that directly - maybe rename to `CredentialsValidator` or `AuthValidator`